src/test/logic/token/utils/tokenHelpers.test.ts

```diff
@@ -0,0 +1,193 @@
+    // given
+    const transaction = getMockedTxServiceModel({ to: safeAddress, value: '0', data: '0xa9055cbb' })
+    const expectedResult = false
+    // when
+    const result = isTokenTransfer(transaction)
+
+    // then
+    expect(result).toEqual(expectedResult)
+  })
+  it('It should return false if the transaction has empty data',  () => {
+    // given
+    const transaction = getMockedTxServiceModel({ to: safeAddress, value: '0', data: null })
+    const expectedResult = false
+    // when
+    const result = isTokenTransfer(transaction)
+
+    // then
+    expect(result).toEqual(expectedResult)
+  })
+})
+
+jest.mock('src/logic/tokens/store/actions/fetchTokens')
+jest.mock('src/logic/contracts/generateBatchRequests')
+jest.mock('console')
+describe('getERC20DecimalsAndSymbol', () => {
+  afterAll(() => {
+    jest.unmock('src/logic/tokens/store/actions/fetchTokens')
+    jest.unmock('src/logic/contracts/generateBatchRequests')
+    jest.unmock('console')
+  })
+  it('It should return DAI information from the store if given a DAI address', async () => {
+    // given
+    const tokenAddress = "0x5592EC0cfb4dbc12D3aB100b257153436a1f0FEa"
+    const decimals = Number(18)
+    const symbol = "DAI"
+    const token = makeToken({
+      address: tokenAddress,
+      name: "Dai",
+      symbol,
+      decimals,
+      logoUri: "https://gnosis-safe-token-logos.s3.amazonaws.com/0x5592EC0cfb4dbc12D3aB100b257153436a1f0FEa.png",
+      balance: 0
+    })
+    const expectedResult = {
+      decimals,
+      symbol
+    }
+
+
+    const fetchTokens = require('src/logic/tokens/store/actions/fetchTokens')
+    const spy = fetchTokens.getTokenInfos.mockImplementationOnce(() => token)
+
+    // when
+    const result = await getERC20DecimalsAndSymbol(tokenAddress)
+
+    // then
+    expect(result).toEqual(expectedResult)
+    expect(spy).toHaveBeenCalled()
+  })
+  it('It should return default value decimals: 18, symbol: UNKNOWN if given a token address and if there is an error fetching the data', async () => {
+    // given
+    const tokenAddress = "0x5592EC0cfb4dbc12D3aB100b257153436a1f0FEa"
+    const decimals = Number(18)
+    const symbol = "UNKNOWN"
+
+    const expectedResult = {
+      decimals,
+      symbol
+    }
+
+
+    const fetchTokens = require('src/logic/tokens/store/actions/fetchTokens')
+    const spy = fetchTokens.getTokenInfos.mockImplementationOnce(() => {throw new Error()})
+    console.error = jest.fn();
+    const spyConsole = jest.spyOn(console, 'error').mockImplementation()
+
+    // when
+    const result = await getERC20DecimalsAndSymbol(tokenAddress)
+
+    // then
+    expect(result).toEqual(expectedResult)
+    expect(spy).toHaveBeenCalled()
+    expect(spyConsole).toHaveBeenCalled()
+  })
+  it('It should fetch token information from the blockchain if given a token address and if the information is not stored', async () => {
```

Comment:
`It should fetch token information from the blockchain if given a token address and if the token doesn't exist in redux store`